core/src/main/java/org/kiwix/kiwixmobile/core/dao/NewNoteDao.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.core.dao
+
+import io.objectbox.Box
+import io.objectbox.kotlin.query
+import io.reactivex.Flowable
+import org.kiwix.kiwixmobile.core.dao.entities.NotesEntity
+import org.kiwix.kiwixmobile.core.dao.entities.NotesEntity_
+import org.kiwix.kiwixmobile.core.page.adapter.Page
+import org.kiwix.kiwixmobile.core.page.notes.adapter.NoteListItem
+import javax.inject.Inject
+
+class NewNoteDao @Inject constructor(val box: Box<NotesEntity>) : PageDao {
+  fun notes(): Flowable<List<Page>> = box.asFlowable(box.query {
```

Comment:
Can we write kotlin's flow instead of Rxjava? We have a ticket that's purpose is   remove all of the Rxjava's Flowable.

Reply:
Sure thing. This can be done here.

Reply:
This cannot be achieved right now, `PageDao` interface uses the `Flowable` changing that will increase the scope of this PR